surf.deckr.deckr.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+    Generator for metainfo & .desktop files:
+https://www.freedesktop.org/software/appstream/metainfocreator/#/
+-->
+<component type="desktop-application">
+  <id>surf.deckr.deckr</id>
+  <name>Deckr</name>
+  <summary>Deckr Desktop App</summary>
+  <developer_name>Steve Gehrman</developer_name>
+  <url type="homepage">https://deckr.surf</url>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+  </supports>
+  <description>
+    <p>Desktop app used with the Deckr chrome plugin. Deckr is a bookmark manager which allows you to share and download bookmark "decks" with others online.</p>
+    <p>Features</p>
```

Comment:
Missing provides/replaces https://docs.flathub.org/docs/for-app-authors/maintenance#renaming-the-flatpak-id

Reply:
Your docs are terrible.  What do I add for provides and replaces?  I can't make any sense of your docs.

Reply:
do I use the old id?  the new ID?  which is provides and which is replaces?

Reply:
It's pretty clear

> please add the old [ID tag](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#id) as a [provides tag](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#provides) and a [replaces tag](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#replaces) in the [MetaInfo file](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#path-and-filename).